app/src/Application/DbSettingsObj.cpp

```diff
@@ -0,0 +1,74 @@
+#include "DbSettingsObj.h"
+
+using namespace Status::Application;
+
+DbSettingsObj::DbSettingsObj(StatusGo::Settings::SettingsDto rawData)
```

Comment:
@MishkaRogachev I name id like that cause it's an object (QObject) exposed to qml, and it will have a lot of props at the end (but read only props) that's why I added "Obj" suffix, and "Db" prefix is there to differ settings stored in the db over settings which will be locally stored in .ini file.

Reply:
@stefandunca I am open for any suggestion on variable naming.